ftplugin/asciidoc.vim

```diff
@@ -375,4 +380,26 @@ nnoremap <buffer> <silent> <LocalLeader>dsb :call asciidoc#experimental#delete_s
 
 " }}}
 
+" Tabular ==================================================          {{{
+if exists(':Tabularize')
```

Comment:
Have you thought about the options here?
This might be the right choice.
First thought would be `exists('g:tabular_loaded')`.
But if I have the plugin on path and don't want to load it for whatever reason, I would set that to prevent the plugin from loading.
Have you given thought to doing this particular check, or is it an "it's good enough" kind of thing?